python/Pipfile

```diff
@@ -0,0 +1,14 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+momento = {editable = true, path = "./../../client-sdk-python"}
```

Comment:
dont forget to update local path to actual version.

Reply:
yup.  need to get a release of the python sdk out first.